sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,39 @@
+function fetchAsyncSignals(mode, customFunction, customKey) {
+    console.log("Going to fetch signals");
+    var eids = "";
+    switch (mode) {
+        case 1:
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 8 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)